### ISSUE TITLE
Add an Installation Guide to the FAQ

### ISSUE THREAD

**Issue by Sherlouk** [SwiftPackageIndex/SwiftPackageIndex-Server#439]
For libraries it may be nice to have an installation guide? This could be a modal to keep the main page as simple as possible.

There is two main methods we'd probably want to add instructions for - via Xcode and via the Package.swift

```swift
// Example
.package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.2.0"))
```

From a UI perspective we'd want to make the URLs easy to copy and paste.

It may be nice to have a step-by-step guide for Xcode - uncertain on how to present it nicely without making it super dense.

**Comment by daveverwer** [comment]
Yea I thought about this when designing the page. I think what would be better than having this repeated on every page is to have a little (?) icon next to the git URL that links to a Markdown page with information on how to use it in a package.

Eventually, most people will know how to integrate a package with their project, and I'd like to keep the page really focused on making the decision about what packages are best for them.

**Comment by daveverwer** [comment]
I think this is fixed by #1519
